api/v1alpha1/openapi.yaml

```
@@ -2426,6 +2470,12 @@ paths:
                 $ref: '#/components/schemas/Error'
 components:
   schemas:
+    DeviceDesiredDecomState:
```

```suggestion
    DeviceDecommission:
      type: object
      properties:
      ...
```

What other minimal information would be useful here? What about a  `reason`? Do we need to know who it was `requestedBy` or is the auth enough? Should we consider this an object for future extensibility?

---

I think we can leave in an open-ended `reason` field and then `requestedBy` would be filled in by the system later (which I think means we don't need it as part of this schema).

---

@fzdarsky suggests not adding anything we don't absolutely need as nothing can be removed later. So I've taken these extra properties back out, while leaving this as an extensible `object`.